cli/admin/src/main/java/io/pravega/cli/admin/bookkeeper/BookkeeperDeleteLedgersCommand.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.bookkeeper;
+
+import io.pravega.cli.admin.CommandArgs;
+import lombok.Cleanup;
+import lombok.val;
+
+/**
+ * This command delete all the ledgers of the log starting with and including given ledger-id.
+ * It can be used as last resort to recover the log in case recovery fails with missing ledger(s).
+ */
+public class BookkeeperDeleteLedgersCommand  extends BookKeeperCommand {
+
+    public BookkeeperDeleteLedgersCommand(CommandArgs args) {
+        super(args);
+    }
+
+    @Override
+    public void execute() throws Exception {
+        ensureArgCount(2);
+        int logId = getIntArg(0);
+        long startId = getLongArg(1);
+
+        @Cleanup
+        val context = createContext();
+        @Cleanup
+        val log = context.logFactory.createDebugLogWrapper(logId);
+
+        // Display a summary of the BookKeeperLog.
+        val m = log.fetchMetadata();
+        if (m == null) {
+            output("BookKeeperLog '%s' does not exist.", logId);
+            return;
+        }
+        outputLogSummary(logId, m);
+
+        log.markAsdisabled();
```

Comment:
Nit: use proper camel case naming for this method `markAsDisabled()`.

Reply:
fixed it